workspaces/global-header/packages/app/src/components/Root/Root.tsx

```diff
@@ -42,66 +37,85 @@ import {
 } from '@red-hat-developer-hub/backstage-plugin-global-header';
 import { NotificationsSidebarItem } from '@backstage/plugin-notifications';
 
-const useSidebarLogoStyles = makeStyles({
-  root: {
-    width: sidebarConfig.drawerWidthClosed,
-    height: 3 * sidebarConfig.logoHeight,
-    display: 'flex',
-    flexFlow: 'row nowrap',
-    alignItems: 'center',
-    marginBottom: -14,
+const useStyles = makeStyles(() => ({
+  pageWithoutFixHeight: {
+    '> div[class*="-sidebarLayout"]': {
+      height: '100vh',
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    '> div > main': {
+      height: 'unset',
+      flexGrow: 1,
+    },
+    '.techdocs-reader-page > main': {
+      height: 'unset',
+    },
   },
-  link: {
-    width: sidebarConfig.drawerWidthClosed,
-    marginLeft: 24,
+  sidebarItem: {
+    textDecorationLine: 'none',
   },
-});
+  sidebarLayout: {
+    '& div[class*="BackstageSidebar-drawer"]': {
+      top: 'var(--rhdh-v1-header-height, 64px)',
+      height: 'calc(100vh - var(--rhdh-v1-header-height, 64px))',
+    },
+    '& main[class*="BackstagePage-root"]': {
+      height: `calc(100vh - (var(--rhdh-v1-header-height, 64px) - var(--rhdh-v1-page-inset, 1.5rem)))`,
+      marginTop: 'calc(-1 * var(--rhdh-v1-page-inset, 1.5rem))',
+      marginBottom: 'calc(-1 * var(--rhdh-v1-page-inset, 1.5rem))',
```

Comment:
I wonder if these styles belong in the theme instead of the root..

Reply:
Hey @logonoff , thanks for the review. I don't know how it looks in our RHDH with the latest theme plugin. I only focused on this dev app. Have you tried the latest theme plugin in the RHDH main? If the issues also exist in there as well, then definitely need a ticket to put these styles in the theme plugin.

Reply:
Oh, by the way, I'm using a fixed header height here as it's only a dev app. In our RHDH the header depends on user's configurations, and the height will be calculated.

Reply:
this is an issue in RHDH as well:

![Uploading image.png…]()

Reply:
https://github.com/redhat-developer/rhdh-plugins/pull/834

Reply:
Hi @logonoff , I have updated this PR to use the latest theme plugin but we still need these styles in the dev app, otherwise it looks off. The dev app dom tree has a different structure than the RHDH, so these seasoning is needed in this PR.